api/valor_api/backend/metrics/classification.py

```diff
@@ -930,4 +931,11 @@ def compute_clf_metrics(
             columns_to_ignore=["value"],
         )
 
+    log_evaluation_analytics(
```

Comment:
I think would prefer the call to this log creation at job creation not computation time. Open to your thoughts on this!

Reply:
This could follow the pattern that were starting of moving eval validators to `backend/core/evaluation.py`

Reply:
discussed why this isn't preferable via slack